### ISSUE TITLE
Confirm 36 videos identified as accessible 

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#1809]

**Comment by mcharg** [comment]
We have 58 videos on the current site, there are 63 in the YouTube Channel, Bills numbers add up to 51, and there are 59 in Bill's spreadsheet. We need to come to consensus on the universe of videos.

**Comment by mcharg** [comment]
There are only 16 videos marked as accessible in Bill's spreadsheet. 7 are marked as not accessible. 33 have no accessibility status.

**Comment by mcharg** [comment]
There are 7 videos on the current site that aren't in Bill's list.

**Comment by Maroyafaied** [comment]
I let Eva know that @mcharg did an inventory and it doesn't match Bill's spreadsheet and that in his spreadsheet only 16 videos were marked as accessible and not 36 as mentioned in previous meetings. I also let her know that @mcharg audited a couple of the videos identified as accessible and found accessibility issues. As of right now , Eva's teams does not have a timeline for this project so I communicated that we will only provide links to the youtube chanel when we launch the new website and not embed videos in webpages.
